File: solutions/python/darts/1/darts.py
""" Module docstring """
import math
def score(x, y):
    """ Module docstring """
    if math.sqrt(x * x + y * y) <= 1:
        return 10
    elif math.sqrt(x * x + y * y) <= 5:
        return 5
    elif math.sqrt(x * x + y * y) <= 10:
        return 1

    return 0

File: solutions/c/darts/1/darts.c
#include "darts.h"
#include <math.h>

uint8_t score (coordinate_t landing_position){
    if (distance(landing_position) > 10.0F) return 0;
    if (distance(landing_position) > 5.0F) return 1;
    if (distance(landing_position) > 1.0F) return 5;
    return 10;
}
float distance(coordinate_t landing_position) {
    float d = sqrt(landing_position.x * landing_position.x + 
                landing_position.y * landing_position.y);
    return d;            
}   
